flink-core/src/test/java/org/apache/flink/api/common/io/GlobFilePathFilterTest.java

```diff
@@ -27,14 +27,13 @@
 import java.io.IOException;
 import java.util.Collections;
 
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+import static org.assertj.core.api.Assertions.assertThat;
 
 public class GlobFilePathFilterTest {
     @Test
```

Comment:
Using assertj is great, would you like to migrate this class to junit5 at the same time?

Reply:
```suggestion
    @Test -> @org.junit.jupiter.api.Test;
```

Reply:
Thanks for your suggestion, I will fix it.